.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig: http://EditorConfig.org
```

Comment:
Configs are generally gitignored in other @airswap repos. If you'd like to commit this here let's discuss with the group